### ISSUE TITLE
./uselib can't open custom networks

### ISSUE THREAD

**Issue by laclouis5** [AlexeyAB/darknet#4479]
I'm using the latest repo with `LIBSO=1` (as well as `GPU`, `CUDNN` and `OPENCV`) and make went fine, there are `darknet`, `uselib` and `libdarknet.so` at yolo root directory.

When running:

`LD_LIBRARY_PATH=./:$LD_LIBRARY_PATH ./uselib data/obj.names custom_model.cfg custom_best.weights test.mov`

I got an error because Yolo is trying to load Yolov3 weights and cfg. Is there a mistake in the command I issued? `argc=0` so it seems that no args are parsed.

I found that those defaults values are hard coded in `yolo_console_dll.cpp` in the main loop, but they should be overridden by the values I entered in my command?

All is ok if I change defaults values to my custom cfg and weights files and recompile.

Last remark, I found that `LD_LIBRARY_PATH` is empty in my case so it just gets initialized to `./`.

**Comment by AlexeyAB** [comment]
Did you try?
`./uselib data/obj.names custom_model.cfg custom_best.weights test.mov`

> argc=0 so it seems that no args are parsed.

As I know the minimum value  that can be `argc=1` if you used `./uselib` without command params.

Did you change anything in the source code?

**Comment by laclouis5** [comment]
> Did you try?
> `./uselib data/obj.names custom_model.cfg custom_best.weights test.mov`
> 
Yes I tried this (from Yolo root dir) but got an error `./uselib: error while loading shared libraries: libdarknet.so: cannot open shared object file: No such file or directory`. I found the `LD_LIBRARY_PATH=./:$LD_LIBRARY_PATH` trick in another related issue.
>
> > argc=0 so it seems that no args are parsed.
> 
> As I know the minimum value that can be `argc=1` if you used `./uselib` without command params.
> 
> Did you change anything in the source code?
>
Yes `argc` should be 5 in my case.
This is strange I just recompile a third time with a debug line to print `argc` and it printed 5 and worked correctly. I think the error was on my side, maybe I did an error in an argument or I forgot the filename for test video.
